installer_scripts/install-yb-voyager

```diff
@@ -523,9 +536,9 @@ rebuild_voyager_local() {
 get_passed_options() {
 	if [ "$1" == "linux" ]
 	then
-		OPTS=$(getopt -o "lpv", --long install-from-local-source,only-pg-support,rebuild-voyager-local --name 'install-yb-voyager' -- $ARGS_LINUX)
+		OPTS=$(getopt -o "lpvV", --long install-from-local-source,only-pg-support,rebuild-voyager-local,version: --name 'install-yb-voyager' -- $ARGS_LINUX)
```

Comment:
is this `lpvV` the args of this script?, if so then as you have changed the `v` -> `r` this needs to be changed to `lprV` right?

Reply:
I havent changed anything old. I just added a new arg 'V' corresponding to the long form "version"
The older 'v' corresponds to the 'rebuild-voyager-local'. It doesnt match with it but i did not change it given that it might be getting used in some automation.